### DISCUSSION TITLE
Weather radar

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#5176] [General]
Hi dev Team,

seems to be, that the weather radar is not working anymore. Started from KMSY to MPTO with thunderstorms (FS 2020 live weather) on the way. Nothing is displayed in the ND. I see flashlights and dark clouds. I can t imagine that there should be no water inside :-p ...

![grafik](https://user-images.githubusercontent.com/76531893/122724614-6788f400-d274-11eb-8b5b-bce8f174c803.png)

Greets Klaus

**Comment by tracernz** [comment]
It works at least sometimes. In any case, this is an asobo bug that we can’t control. There is no way for us to retrieve the weather information to draw our own, so we can only use the image supplied by the sim.
